README.md

```diff
@@ -152,34 +152,37 @@ If you have any problems building, please consult the developer [FAQ][].
 
 ### Build status of master branches
 
-| AppVeyor (Windows)       | Travis CI (Linux / macOS) | VSTS (Windows) | VSTS (Linux) | VSTS (macOS) |
-|--------------------------|---------------------------|----------------|--------------|--------------|
-| [![av-image][]][av-site] | [![tv-image][]][tv-site]  |[![vsts-windows-image][]][vsts-windows-site] | [![vsts-linux-image][]][vsts-linux-site]  | [![vsts-macos-image][]][vsts-macos-site] |
+| AppVeyor (Windows)       | Azure (Windows)                          | Azure (Linux)                        | Azure (macOS)                        | Azure (spelling)                     |
+|:-------------------------|:-----------------------------------------|:-------------------------------------|:-------------------------------------|:-------------------------------------|
+| [![av-image][]][av-site] | [![az-windows-image][]][az-windows-site] | [![az-linux-image][]][az-linux-site] | [![az-macos-image][]][az-macos-site] | [![az-spell-image][]][az-spell-site] |
 
 ### Build status of nightly builds
 
-| AppVeyor (Windows)       | Travis CI (Linux) | Travis CI (macOS) | Code Coverage Status | Code Factor Status |
-|--------------------------|-------------------|-------------------|----------------------|--------------------|
-| [![av-nightly-image][]][av-nightly-site] | [![linux-nightly-image][]][tv-site] | [![macOS-nightly-image][]][tv-site] | [![cc-image][]][cc-site] | [![cf-image][]][cf-site] |
+| AppVeyor (Windows)                       | Azure CI (Linux)                               | Azure CI (macOS)                               | Code Coverage Status     | Code Factor Status       |
```

Comment:
minor: maybe just `Azure (Linux)` and `Azure (macOS)` without `CI`? `CI` was from `Travis CI`.